configure.ac

```diff
@@ -434,6 +434,13 @@ if test x"$with_openssl" = xno ; then
     AC_MSG_ERROR([This release of CFEngine requires OpenSSL >= 0.9.7])
 fi
 
+if test -d /usr/local/Cellar/openssl && test "x$with_openssl" = "xyes" ; then
+    openssl_include=( $(find /usr/local/Cellar/openssl/1*/ -name include -type d | sed -n 1p) )
```

Comment:
I'm not familiar with autoconf scripts, but I guess there is a reason for using braces instead of quotes? I.e. `varname=( $(...) )` instead of `varname="$(...)"`